### PR TITLE
Add sections works, about me, contact form, footer

### PR DESCRIPTION
**Pull request by s2jack**
Add works section
Add about me section
Add contact form
Add footer
Fix linters

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -198,3 +200,532 @@ a {
...
+
+#works {
+  display: flex;
+  position: absolute;
+  top: 708px;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 3283px;
+}
```
Comment (Tufahel) [inline styles.css:234]: - [ ] This section is the main container of the work `section`. The main container should have `display: grid`. This means all cards should be arranged using `grid` not `flexbox`. Kindly fix this issue.

File: styles.css [hunk 2]
```diff
@@ -198,3 +200,532 @@ a {
...
+  height: 69px;
+}
+
+#pr-detail-btn {
+  width: 122px;
+  height: 48px;
+  font-size: 17px;
+  line-height: 24px;
+  font-weight: 500;
+}
```
Comment (Tufahel) [inline styles.css:378]: - [ ] Kindly fix the button color. Implement the button interactions (enable, hover, etc.) also to match the given design.
- image

**Review by Tufahel — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @s2jack :wave:  

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights 🌟

Alt Text

✔️ Descriptive pull request.
✔️ Linter checks are passing.
✔️ Follow the correct GitHub flow.
✔️ Commit messages are meaningful.

## Required Changes ♻️

***Please kindly check the comments/review below.***

## Optional suggestions 🤷‍♂️

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (s2jack) [reply styles.css:234]: I will share the instruction screenshot. So you can see that is not mandatory. This is my works section container that's right. But the instruction tells the main container must have grid attribute that contains all of your cards. So my cards container has grid attribute. Please take a look of my codes again. By the way, thanks for your time for all of your revisions.

File: index.html [hunk 3]
```diff
@@ -8,43 +8,220 @@
...
+        <div id="cards-cont">
+          <div class="card" id="card1">
+            <div class="project-info" id="project1">
+              <h3 class="project-name">Multi-Post Stories Gain+Glory</h3>
+              <ul class="pr-tags-ul">
+                <li class="pr-tag">Ruby on Rails</li>
+                <li class="pr-tag">CSS</li>
+                <li class="pr-tag">JavaScript</li>
+                <li class="pr-tag">HTML</li>
+              </ul>
```
Comment (AbayomiOlaoye) [inline index.html:61]: - [x] [OPTIONAL] While this current `color` looks good and without much impact on user experience, the expected color for the text is `color: #3a4a42` not `#000`. Apart from matching the design a little closer, I think it's more aesthetically impressive :+1:

**Review by AbayomiOlaoye — APPROVED** [review]
## STATUS: APPROVED :green_circle:  

Hi @s2jack :wave:   :wave: :wave:, 

Sorry about the previous misconception of your `works-section` structure, and thanks to him, you were able to make your portfolio a little more visually appealing :smile: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

applause

## Highlights
- Cleanly written code :white_check_mark: 
- Good response to feedback ✅ 
- Nicely implemented UI styles ✅ 
- Button states are nicely styled ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 1 above)
Comment (Tufahel) [reply styles.css:234]: - Hi, @s2jack 👋 
- Sorry for the misconception. I can see you are right.
- Happy Coding 😄

**Pull request merged**